tests/utils/test_validation.go

```diff
@@ -26,6 +26,18 @@ func ValidateXrayVersion(t *testing.T, minVersion string) {
 	}
 }
 
+func ValidateXscVersion(t *testing.T, minVersion string) {
+	xscVersion, err := getXscVersion()
+	if err != nil {
+		assert.NoError(t, err)
+		return
+	}
+	err = clientUtils.ValidateMinimumVersion(clientUtils.Xsc, xscVersion.GetVersion(), minVersion)
+	if err != nil {
+		t.Skip(err)
+	}
```

Comment:
```suggestion
	if err = clientUtils.ValidateMinimumVersion(clientUtils.Xsc, xscVersion.GetVersion(), minVersion); err != nil {
		t.Skip(err)
	}
```

Reply:
The way I implemented it is totally fine as well.